src/runner.py

```diff
@@ -52,6 +53,9 @@
 
 Snap = NamedTuple("Snap", [("name", str), ("channel", str)])
 
+YQ_BIN_URL_AMD64 = "https://github.com/mikefarah/yq/releases/download/v4.34.1/yq_linux_amd64"
+YQ_BIN_URL_ARM64 = "https://github.com/mikefarah/yq/releases/download/v4.34.1/yq_linux_arm64"
```

Comment:
Please be reminded that the binaries download is being reworked in another feature branch and will be removed.